bin/analysercli

```diff
@@ -0,0 +1,262 @@
+#! /usr/bin/env python
+#
+# This file is part of IVRE.
+# Copyright 2011 - 2015 Pierre LALET <pierre.lalet@cea.fr>
+#
+# IVRE is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# IVRE is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+# or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public
+# License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with IVRE. If not, see <http://www.gnu.org/licenses/>.
+
+from ivre import webutils
+from ivre.db import db
+from ivre.cliutils import colorize_log, StackCli
+from ivre.analyser import Context
+
+
+class AnalyserCli(StackCli):
+    """
+    Command line utility wrapper to access analyser API
+    """
+    def __init__(self):
+        StackCli.__init__(self)
+        self._dfilter = {}
+
+    def preloop(self):
+        context = Context()
+        colorize_log('info', 'Initialization')
+        self._build_filter(raw_input('Filter > '))
+        colorize_log('info', 'Reading database..')
+        context.dfilter = self._dfilter
+        context.init_hosts()
+        self.push(context)
+
+    def postloop(self):
+        colorize_log('info', 'Leaving..')
+
+    @staticmethod
+    def flt_param(param):
+        """Search for parameter starting with '!'"""
```

Comment:
1. The function name and the docstring are not really consistent with what the function actually does.
2. If this function is only used from a "private" method, it should probably also be "private", or have I missed something?
3. Please avoid unneeded variables:
```Python
if param.startswith('!'):
    return True, param[1:]
return False, param
```

Or better:
```Python
return (True, param[1:]) if param.startswith('!') else (False, param)
```